src/bin/vip-logs.js

```diff
@@ -232,31 +232,39 @@ command( {
 	envContext: true,
 	module: 'logs',
 } )
-	.option( 'type', 'The type of logs to be returned: "app" or "batch"', 'app' )
+	.option(
+		'type',
+		'Specify the type of Runtime Logs to retrieve. Accepts "batch" (only valid for WordPress environments) or "app" (default).'
+	)
```

Comment:
```suggestion
	.option(
		'type',
		'Specify the type of Runtime Logs to retrieve. Accepts "batch" (only valid for WordPress environments) or "app" (default).',
              'app'
	)
```

Reply:
It's hard to guess the indentation from the phone :)

The idea is that if you don't specify the default value, the app will force you to provide it.

Reply:
See https://github.com/Automattic/vip-cli/commit/5e1cd804cc6dd715309536a533a647d78f21e6d5

Reply:
> Does that argument do something other than populate that appended (Defaults to <argument>) phrase?

Unfortunately, yes. If the parameter is not provided on the command line, it is artificially created with that default value.

Compare the output of `vip logs -l 1` with and without the default argument.

![Screenshot_20241121_202847](https://github.com/user-attachments/assets/0ad279a2-c8dc-4042-be0e-72a0dfa7e5bd)

Reply:
If you don't like the "Defaults to..." phrase, I can update the logic to assume the type is `app` if it is not provided on the command line.

Reply:
The same is true for `format`:

```
Error:  Invalid format: undefined. The supported formats are: csv, json, table.
Debug:  VIP-CLI v3.8.8-dev.0, Node v20.17.0, linux 6.8.0-49-generic x64
```

Reply:
Addressed here: https://github.com/Automattic/vip-cli/pull/2089/files#diff-2842aeea5897751928ed910d842cd542768c354d43d746f1877afab1869eac9fR26-R28

Reply:
I believe that I have repaired the damage done. Thank you for catching this and explaining!